1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,7 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy = store.filter((element) => element.price < money).map((element) => element.item);
```

Comment:
Nice chain 👍 
You can also remove parenthesis around the parameter of the arrow functions, like:
```arr.map(element => element.item)```